### PR TITLE
adding helloworld console app

### PR DESCRIPTION
**Pull request by ARF-DEV**
add hello world app in c++

### CODE REVIEW

File: 119140149.cpp [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+
```
Comment (gremlinflat) [inline 119140149.cpp:2]: kebanyakan new line rip wkwkwk

**Comment by riwandys** [comment]
Nice feature, work well

**Review by AndhikaWB — APPROVED** [review]

File: 119140149.cpp (hunk 1 above)
Comment (ARF-DEV) [reply 119140149.cpp:2]: done

Comment (ARF-DEV) [reply 119140149.cpp:2]: udah

**Pull request merged**